src/sections/ui/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,24 @@
+import { Tooltip as TooltipBS } from 'react-bootstrap'
+import { OverlayTrigger } from 'react-bootstrap'
+import { Placement } from 'react-bootstrap/types'
+import { QuestionIcon } from './QuestionIcon'
+
+export interface TooltipProps {
+  placement: Placement
+  message: string
+}
+
+export function Tooltip({ placement, message }: TooltipProps) {
+  return (
+    <>
+      <OverlayTrigger
+        key={placement}
+        placement={placement}
+        overlay={<TooltipBS id={`tooltip-${placement}`}>{message}</TooltipBS>}>
```

Comment:
`overlay={<TooltipBS id={`tooltip-${placement}`}>{message}</TooltipBS>}>`

If the id is determined only by the placement it could happen that we have 2 equal ids in the same page if we have 2 tooltips with the same placement. Maybe we need to pass the id as a prop to don't have duplicated id's